utils/src/main/kotlin/DiskCache.kt

```diff
@@ -38,9 +38,9 @@ class DiskCache(
         directory: File,
```

Comment:
Commit message nit: Do not start the commit message body with "..." but simply start a new sentence instead of continuing the commit message title.